docker-compose.yaml

```diff
@@ -2,24 +2,28 @@ version: "3.7"
 
 services:
   db:
-    #platform: linux/arm64
-    build: ./mysql
+    build: ./postgres
     container_name: tts_db
     volumes:
-      - ./mysql/data:/var/lib/mysql
+      - postgres_data:/var/lib/postgresql/data/
```

Comment:
Since in the makefile and so on the data folder is `./postgres/data`, shouldn't this be 

`./postgres/data:/var/lib/postgresql/data/` instead of `postgres_data:/var/lib/postgresql/data/`.

I was having a lot of trouble cleaning and repopulating the database and this change fixed it. However, I wanted your confirmation that it should be `./postgres/data` because there could have been another reason I wasn't being able to clean the data

Reply:
Yeah, but postgres doesn't really play well with volumes mounted on your fs. That's why I changed it to a classic volume.

I'll change the Makefile to clean the database using docker compose commands.